<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>665</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Bežalec</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="maximumSize">
       <size>
        <width>908</width>
        <height>588</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Registracija</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="6" column="1">
         <widget class="QComboBox" name="comboBoxRegSpol">
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <item>
           <property name="text">
            <string>Moški</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ženski</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0" rowspan="12">
         <widget class="QTableWidget" name="tableWidgetRegVnos">
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>7</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Štartna številka</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ime in Priimek</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Poklic</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Klub</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Kategorija</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Spol</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Urejanje</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditRegIme">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="placeholderText">
           <string>Ime in Priimek</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Kategorija</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditRegKlub">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="placeholderText">
           <string>Klub</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Spol</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="pushButtonReg">
          <property name="text">
           <string>Vnesi</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="comboBoxRegKat">
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditRegNumber">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="inputMask">
           <string>999</string>
          </property>
          <property name="maxLength">
           <number>3</number>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="cursorPosition">
           <number>0</number>
          </property>
          <property name="placeholderText">
           <string>Štartna številka</string>
          </property>
          <property name="cursorMoveStyle">
           <enum>Qt::LogicalMoveStyle</enum>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Vnos</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0,0,0,0">
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEditCompCas">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="placeholderText">
           <string>Čas (hh:mm:ss)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="pushButtonComp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Vnesi čas</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEditCompStartna">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="placeholderText">
           <string>Tekmovalec</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QTableWidget" name="tableWidgetCompVnos">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="columnCount">
           <number>8</number>
          </property>
          <column>
           <property name="text">
            <string>Štartna številka</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ime in priimek</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Poklic</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Klub</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Kategorija</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Spol</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Čas</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Urejanje</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatoteka">
    <property name="title">
     <string>Datoteka</string>
    </property>
    <addaction name="actionOdpri"/>
    <addaction name="actionShrani"/>
    <addaction name="actionIzhod"/>
   </widget>
   <widget class="QMenu" name="menuNastavitve">
    <property name="title">
     <string>Nastavitve</string>
    </property>
    <addaction name="actionKategorije"/>
   </widget>
   <widget class="QMenu" name="menuIzpis">
    <property name="title">
     <string>Izpis</string>
    </property>
    <addaction name="actionPripravi_izpis"/>
   </widget>
   <addaction name="menuDatoteka"/>
   <addaction name="menuNastavitve"/>
   <addaction name="menuIzpis"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOdpri">
   <property name="text">
    <string>Odpri</string>
   </property>
  </action>
  <action name="actionShrani">
   <property name="text">
    <string>Shrani</string>
   </property>
  </action>
  <action name="actionIzhod">
   <property name="text">
    <string>Izhod</string>
   </property>
  </action>
  <action name="actionKategorije">
   <property name="text">
    <string>Kategorije</string>
   </property>
  </action>
  <action name="actionPripravi_izpis">
   <property name="text">
    <string>Pripravi izpis</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
